### PR TITLE
Add `mixinexport` tag

### PR DESCRIPTION
**Pull request by dicedpixels**
Exporting Mixins seem to come up quite often. This can be a quick reference. `mixindebug.ytag` was removed as the new one covers that as well.

### CODE REVIEW

File: tags/guide/mixinexport.ytag [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Example:
+```java
+@Debug(export = true)
+@Mixin(...)
+public class MyMixin {
+    // Mixin code here
+}
+```
+You can export **all** Mixin classes by adding `-Dmixin.debug.export=true` to your VM options.
```
[the remark this answers is not in the document]
Comment (dicedpixels) [reply tags/guide/mixinexport.ytag:21]: There's a guide? I can add that.

File: tags/guide/mixinexport.ytag [hunk 2]
```diff
@@ -0,0 +1,25 @@
+type: embed
+
+colour: black
+embed:
+    title: Exporting Mixin Classes
+    image:
+        url: "https://cdn.discordapp.com/attachments/523633816078647296/1176733253877641306/mixin-export.png"
```
Comment (modmuss50) [inline tags/guide/mixinexport.ytag:7]: Can you move this image to the repo: https://github.com/FabricMC/community/tree/main/images

Comment (dicedpixels) [reply tags/guide/mixinexport.ytag:7]: Sure! Although there are several other tags with Discord CDN images. eg. cheatsheet/sides.ytag

**Review by apple502j — APPROVED** [review]

**Comment by Benonardo** [comment]
Might be worth to mention searching for `@MixinMerged` here?

**Comment by dicedpixels** [comment]
> Might be worth to mention searching for `@MixinMerged` here?

Sure. What's a good way to word it?

**Pull request merged**